src/components/backend-ai-painkiller.ts

```diff
@@ -51,7 +51,7 @@ export default class BackendAIPainKiller {
     'integrity error: duplicate key value violates unique constraint "pk_scaling_groups"[\\n]DETAIL:  Key \\(name\\)=\\([\\w]+\\) already exists.[\\n]': 'error.ScalingGroupAlreadyExist',
     'integrity error: duplicate key value violates unique constraint "uq_users_username"[\\n]DETAIL:  Key \\(username\\)=\\([\\w]+\\) already exists.[\\n]': 'error.UserNameAlreadyExist',
     'server responded failure: 400 Bad Request - Missing or invalid API parameters. (Your resource quota is exceeded. (cpu=24 mem=512g cuda.shares=80))': 'error.ResourceLimitExceed',
-    'integrity error: update or delete on table "keypair_resource_policies" violates foreign key constraint "fk_keypairs_resource_policy_keypair_resource_policies" on table "keypairs"[\\n]DETAIL:  Key \\(name\\)=\\([\\w]+\\) is still referenced from table "keypairs".[\\n]': 'error.ResourcePolicyStillReferenced',
+    'integrity error: \\(sqlalchemy.dialects.postgresql.asyncpg.IntegrityError\\) \\<class \'asyncpg.exceptions.ForeignKeyViolationError\'\\>: update or delete on table "keypair_resource_policies" violates foreign key constraint "fk_keypairs_resource_policy_keypair_resource_policies" on table "keypairs"[\\n]DETAIL:  Key \\(name\\)=\\([\\w]+\\) is still referenced from table "keypairs".[\\n]\\w*': 'error.ResourcePolicyStillReferenced',
```

Comment:
Thank you for your contribution.
It didn't work. Let's make a shorter regexp,
I think that checking only `Key \(name\)=\(resoure-policy\) is still referenced from table "keypairs` is quite enough.

Reply:
I modified code and requested review again. Also I tried this code, It worked well🧐
Please check one more time😊